### PR TITLE
Styles index

### PR DESCRIPTION
**Pull request by philjdelong**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <br>
 
       <div align='center' class="link">
-        <%= link_to "Styles"%>
+        <%= link_to "Styles", "/styles" %>
```
Comment (wrenisit) [inline app/views/layouts/application.html.erb]: I like the change here

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
+require "simplecov"
+SimpleCov.start
+
+require_relative './factories/factory_bot'
```
Comment (wrenisit) [inline spec/rails_helper.rb]: Thank you for setting up factory bot.

**Comment by philjdelong** [comment]
Travis is my least favorite

**Comment by LainMcGrath** [comment]
Thanks for making these changes, Phil! Excited to get move forward with this functionality.

Build wasn't failing but was running for more than three hours. I added `config.include FactoryBot::Syntax::Methods` to the rails_helper.rb file and removed the factory_bot.rb. Travis was able to complete the build. Merging into master!

**Pull request merged**